www/src/components/pdf/ViewerSDKClient.ts

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/*
+Copyright 2020 Adobe
+All Rights Reserved.
+
+NOTICE: Adobe permits you to use, modify, and distribute this file in
+accordance with the terms of the Adobe license agreement accompanying
+it. If you have received this file from a source other than Adobe,
+then your use, modification, or distribution of it requires the prior
+written permission of Adobe.
+*/
```

Comment:
Any specific reason this is here? Is it required :rofl:

Reply:
loool I just kept it there for no reason. I kinda wanna keep it there to kinda credit Adobe that we're using their code? Idrk how that works though (i.e. properly credit Adobe for using their code), is it sufficient to remove it and just say like "Code adopted from Adobe example code"?

Reply:
Eh, probably fine to just keep it there I guess